docs/src/main/asciidoc/logging.adoc

```diff
@@ -54,25 +139,16 @@ The root logger category is handled separately, and is configured via the follow
 |quarkus.log.level|INFO|The default minimum log level for every log category.
 |===
 
-=== Log levels
+The root logger configuration is used when no configuration exists for a given logger category.
 
-There are several log levels you can use:
+== Logging Format
 
-[cols="<m,<5",options="header"]
-|===
-|Level|Description
-|FATAL|A critical service failure/complete inability to service requests of any kind
-|ERROR|A significant disruption in a request or the inability to service a request
-|WARN|A non-critical service error or problem that may not require immediate correction
-|INFO|Service lifecycle events or important related very-low-frequency information
-|DEBUG|Messages that convey extra information regarding lifecycle or non-request-bound events which may be helpful for debugging
-|TRACE|Messages that convey extra per-request debugging information that may be very high frequency
-|===
+By default, Quarkus uses a **String** logging format that format a log event in a simple text.
 
-[id="format-string"]
-== Format String
+You can configure this logging format for each log handler via a dedicated property,
+for the console handler it will be the `quarkus.log.console.format` property.
```

Comment:
```suggestion
You can configure the format for each log handler via a dedicated property.
For the console handler, the property is `quarkus.log.console.format`.
```

Reply:
We should said `You can configure the pattern` but the configuration is called format and we don't have configuration alias to easily change this without breaking compatibility ...

Reply:
That's OK; I think either way is understandable.